fossology/fossology
# same fix for UploadSrvPage.php as for UploadCvsPage.php see #2068

Signed-off-by: Martin Daur <mdaur@gmx.net>



## Description

This fix is about missing postUploadAddJobs in UploadSrvPage.php otherwise plugins like reuse are not run in case of server  uploads (menu item "From Server"). See also #2068.

### Changes

add postUploadAddJobs in UploadSrvPage.php remove duplicate code.

## How to test

perform any upload "From Server" and check if plugins get executed (e.g. reuser, decider etc.). Without this fix these kind of plugins will not be executed.

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)
